.env-template

```diff
@@ -10,3 +10,4 @@ FROM_EMAIL=
 DOMAIN=http://localhost:8000
 ENV=local
 SLACK_VERIFICATION_TOKEN=
+PYBITES_EMAIL_GROUP=bob@pybit.es julian@pybit.es
```

Comment:
@greywidget minor thing, can you please make this a comma separated list, here is an example how we use it for allowed hosts:

```
.env
ALLOWED_HOSTS=.localhost, .herokuapp.com

settings.py
ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
```